### DISCUSSION TITLE
Assignment cutoff date is ineffective

### DISCUSSION THREAD

**Discussion by jheinzel** [community/community#71898] [GitHub Education]
### Hi everyone,

in one of my classrooms, all assignments exhibit an unexpected behavior - they are assigned a deadline as a cutoff date. However, upon reaching this deadline, the students don't lose their write permissions as expected. Therefore, they are still able to their repositories. What could be the cause for this anomaly?

**Comment by georgernstgraf** [comment]
I only see this problem when students push via ssh.

**Comment by jheinzel** [comment]
Thank you for your reply and insight. Your observation seems highly plausible as it echoes my own experience. I have established two GitHub Classrooms in my setting, with one utilizing SSH and the other using HTTP. Interestingly, the issue you mentioned only arises in the GitHub Classroom which uses HTTP.

Given this, I would really appreciate it if anyone has any suggestions on how to circumvent this problem. Any help would be greatly appreciated.

**Comment by maartentroost** [comment]
This is still an issue, even when students use GH desktop.

**Reply by SteijlenGLR** [reply]
Yes, we have still experienced this faulty behaviour. 
Students working in both CodeSpaces and on their local Visual Studio Code pushing code via HTTP and/or SSH still have the option to submit code after the cutoff date.

**Comment by isaiaslafon** [comment]
We have the same issue, students are using Visual Studio Code.
